lazypredict/Supervised.py

```diff
@@ -217,13 +217,15 @@ def __init__(
         custom_metric=None,
         predictions=False,
         random_state=42,
+        classifiers = CLASSIFIERS
```

Comment:
I think that the default should be `classifiers="all"` and then in the line 289 we can do:

```
if self.classifiers == "all":
    self.classifiers = CLASSIFIERS
else:
    **your code**
```

Please add the documentation from this parameter in the class docstring.